src/effects/effectslot.cpp

```diff
@@ -179,8 +181,6 @@ void EffectSlot::loadEffect(EffectPointer pEffect) {
             pParameter->loadEffect(pEffect);
         }
 
-        slotEffectMetaParameter(m_pControlMetaParameter->get(), true);
```

Comment:
> If the effect is disabled, which is recommended when chaining effects, A touch at the meta-knob makes the parameter jump to the linked position

This should not require moving the metaknob. That could be easily accomplished by just removing the `, true` from this line.